integration-tests/contracts/ethereum_keeper_contracts.go

```diff
@@ -1724,25 +1759,113 @@ func (v *EthereumKeeperRegistrar) EncodeRegisterRequest(
 		}
 		return req, nil
 	} else if v.registrar21 != nil {
-		req, err := registrarABI.Pack(
-			"register",
-			name,
-			email,
-			common.HexToAddress(upkeepAddr),
-			gasLimit,
-			common.HexToAddress(adminAddr),
-			uint8(0), // trigger type
-			checkData,
-			[]byte{}, // triggerConfig
-			[]byte{}, // offchainConfig
-			amount,
-			common.HexToAddress(senderAddr),
-		)
+		if isLogTrigger {
+			// bytes representation of 0x3d53a39550e04688065827f3bb86584cb007ab9ebca7ebd528e7301c9c31eb5d
+			topic0InBytes := [32]byte{
+				61, 83, 163, 149, 80, 224, 70, 136,
+				6, 88, 39, 243, 187, 134, 88, 76,
+				176, 7, 171, 158, 188, 167, 235,
+				213, 40, 231, 48, 28, 156, 49, 235, 93,
+			}
 
-		if err != nil {
-			return nil, err
+			topic1InBytes := [32]byte{
+				0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0,
+			}
+
+			topic2InBytes := [32]byte{
+				0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0,
+			}
+
+			topic3InBytes := [32]byte{
+				0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0,
+			}
+
+			logTriggerConfigStruct := automation_utils_2_1.LogTriggerConfig{
+				ContractAddress: common.HexToAddress(upkeepAddr),
+				FilterSelector:  0,
+				Topic0:          topic0InBytes,
+				Topic1:          topic1InBytes,
+				Topic2:          topic2InBytes,
+				Topic3:          topic3InBytes,
+			}
+			encodedLogTriggerConfig, err := utilsABI.Pack("_logTriggerConfig", &logTriggerConfigStruct)
+			if err != nil {
+				return nil, err
+			}
+
+			req, err := registrarABI.Pack(
+				"register",
+				name,
+				email,
+				common.HexToAddress(upkeepAddr),
+				gasLimit,
+				common.HexToAddress(adminAddr),
+				uint8(1), // trigger type
+				checkData,
+				encodedLogTriggerConfig, // triggerConfig
+				[]byte{},                // offchainConfig
+				amount,
+				common.HexToAddress(senderAddr),
+			)
+
+			if err != nil {
+				return nil, err
+			}
+			return req, nil
+
+			//logTriggerConfigType := goabi.MustNewType("tuple(address contractAddress, uint8 filterSelector, bytes32 topic0, bytes32 topic1, bytes32 topic2, bytes32 topic3)")
```

Comment:
I kept this for comparison, PR is not ready for landing.

@RyanRHall  , I am trying to use the utils but with the change, upkeeps dont perform, I believe its bc the topics dont match. I think the byte[32] aligns with the original hex string. Could you help double check?